packages/instrumentation-socket.io/README.md

```diff
@@ -34,7 +34,9 @@ socket.io instrumentation has few options available to choose from. You can set
 | Options        | Type                                   | Description                                                                                     |
 | -------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------- |
 | `emitHook` | `SocketIoHookFunction` | hook for adding custom attributes before socket.io emits the event |
+| `emitIgnoreEventList` | `string[]` | names of emitted events to ignore |
 | `onHook` | `SocketIoHookFunction` | hook for adding custom attributes before the event listener (callback) is invoked |
+| `onIgnoreEventList` | `string[]` | names of listened events to ignore |
```

Comment:
I would also add "ignore tracing" like in the comment on the config class. When read like this it can be a bit unclear what we are ignoring :)